packages/nx/changelog-renderer/index.ts

```diff
@@ -294,11 +312,37 @@ function groupBy(items: any[], key: string) {
 function formatCommit(commit: GitCommit, repoSlug?: RepoSlug): string {
   let commitLine =
     '- ' +
-    (commit.scope ? `**${commit.scope.trim()}:** ` : '') +
     (commit.isBreaking ? '⚠️  ' : '') +
+    (commit.scope ? `**${commit.scope.trim()}:** ` : '') +
```

Comment:
Moved these around it looks better when they line up.

Real example of the "after" of this PR from rxjs repo:

![image](https://github.com/nrwl/nx/assets/900523/43aeb664-3548-4ca5-bbcc-7be59893c476)